telemetry/telemetry_record.py

```diff
@@ -73,15 +100,48 @@ def move_wpr_files(name_schema):
     """
     remote_data_path = os.path.join(CHROMIUM_SRC, 'tools/perf/page_sets/data/',
             name_schema)
-    local_path = os.path.join(os.getcwd(), 'data/')
+    local_path = os.path.join(os.getcwd(), 'data/wpr_source')
     p = Popen('mv {0} {1}'.format(remote_data_path, local_path), shell=True)
 
+def make_modified_json(num_urls):
+    """Creates *.json files to point to the right archive files
+
+    Located in ~/src/tools/perf/page_sets/data
+    :param num_urls: int for the number of target urls
+    TODO: add sha1?
+    """
+    json_path = '/home/jamshed/src/tools/perf/page_sets/data/'
```

Comment:
True, it doesn't seem super necessary. Is there a specific reason for why relative is better than absolute? The [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) style guide seems to indicate that implicit relative imports are worse than explicit or absolute, which is why I made everything absolute. This way it can also be run from anywhere. 

I guess it doesn't make a huge different here, but absolute seems more clear. I'm curious as to other reasonings though!